src/hl/filters.rs

```diff
@@ -121,16 +123,16 @@ pub struct FilterInfo {
 pub(crate) fn register_filters() {
     #[cfg(feature = "lzf")]
     if let Err(e) = lzf::register_lzf() {
-        eprintln!("{}", e);
+        eprintln!("Error while registering LZF filter: {}", e);
     }
     #[cfg(feature = "blosc")]
     if let Err(e) = blosc::register_blosc() {
-        eprintln!("{}", e);
+        eprintln!("Error while registering Blosc filter: {}", e);
     }
 }
 
-/// Returns `true` if gzip filter is available.
-pub fn gzip_available() -> bool {
+/// Returns `true` if deflate filter is available.
+pub fn deflate_available() -> bool {
```

Comment:
Add `gzip_available` but mark as deprecated to keep semver to minor

Reply:
Yea, good point